pipes/WDL/workflows/scaffold_and_refine_multitaxa.wdl

```diff
@@ -79,7 +70,7 @@ workflow scaffold_and_refine_multitaxa {
             "assembly_length_unambiguous" : refine.assembly_length_unambiguous,
             "reads_aligned" : refine.align_to_self_merged_reads_aligned,
             "mean_coverage" : refine.align_to_self_merged_mean_coverage,
-            "percent_reference_covered" : 1.0 * refine.assembly_length_unambiguous / refine.reference_genome_length,
+            "percent_reference_covered" : select_first([percent_reference_covered, 0.0]),
```

Comment:
It may be nice to break out `tax_name` and `percent_reference_covered` for the "top" viral assembly into separate workflow outputs, for easier search and filtering on Terra (where "top" could be defined as the most complete assembly, or the most abundant taxon in terms of # of reads or # of matching distinct k-mers).